src/libs/filters/aperture.c

```diff
@@ -67,7 +67,19 @@ static gboolean _aperture_update(dt_lib_filtering_rule_t *rule)
 
 static gchar *_aperture_print_func(const double value, const gboolean detailled)
 {
-  return g_strdup_printf("%s%.1lf", detailled ? "f/" : "", value);
+  if(detailled)
+  {
+    return g_strdup_printf("%s%.1lf", detailled ? "f/" : "", value);
```

Comment:
But we still can remove `detailled` here as we are already in the conditional `if(detailled)`.

Reply:
ahh, now I see you point :-)
Didn't touch that original line.